ntex/src/connect/rustls.rs

```diff
@@ -37,7 +37,7 @@ impl<T: Address + 'static> RustlsConnector<T> {
         Connect<T>: From<U>,
     {
         let req = Connect::from(message);
-        let host = req.host().to_string();
+        let host = req.host().split(":").next().unwrap().to_owned();
```

Comment:
The split, next, unwrap is inaflible:

    assert_eq!("".split(":").next().unwrap(),"");